### PR TITLE
Minor tweaks to simplify sample configuration.

### PR DESCRIPTION
**Pull request by gguuss**

### CODE REVIEW

File: vision/explore-api/main.js [hunk 1]
```diff
@@ -13,19 +13,17 @@
 
 'use strict';
 
-// Expecting this line in file named key.js
-// window.apiKey = "[YOUR API KEY HERE]"
 var CV_URL = 'https://vision.googleapis.com/v1/images:annotate?key=' + window.apiKey;
 
-$(document).ready(function () {
+$(document).ready(function() {
```
Comment (theacodes) [inline vision/explore-api/main.js]: Could just do `$(function() {` ?

File: vision/explore-api/main.js [hunk 2]
```diff
@@ -13,19 +13,17 @@
...
   $('#fileform').on('submit', uploadFiles);
 });
 
 /**
  * 'submit' event handler - reads the image bytes and sends it to the Cloud
  * Vision API.
  */
-function uploadFiles (event) {
+function uploadFiles(event) {
```
Comment (theacodes) [inline vision/explore-api/main.js]: In node.js, at least, we retain the space after the function name.

File: vision/explore-api/main.js (hunk 1 above)
Comment (gguuss) [reply vision/explore-api/main.js]: Done.

File: vision/explore-api/main.js (hunk 2 above)
Comment (gguuss) [reply vision/explore-api/main.js]: Done. Sample should now pass semistandard lint.

**Review by theacodes — APPROVED** [review]
This LGTM, but I'm concerned about Travis failing. @tswast, any ideas?

**Comment by tswast** [comment]
Looks like it's trying to launch Firefox. Strange because I configured it to use the "karma-detect-browsers" plugin, which should only run against PhantomJS on Travis.

I think the "fix" would be to explicitly declare the phantomjs runner https://github.com/karma-runner/karma-phantomjs-launcher and maybe have commented out Chrome, Firefox, Safari, etc. for local testing.

**Comment by tswast** [comment]
Oh, looks like I don't extract a key.js file in .travis.yml, either. I think I meant to do that when I created the repo, but forgot.

**Review by tswast — APPROVED** [review]
Thanks, LGTM. I'll fix Travis to extract an encrypted API key in another PR.

**Pull request merged**